cypress/integration/login.js

```diff
@@ -21,7 +21,11 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Ugne Lekstutiene');
         cy.get('.main-nav').find('li').should('have.length', 1);
     })
+
+    it('Date today', function()  {
+        cy.get('.calendar--today').contains('24'); 

```

Comment:
This test case is very depended on a previous one, if previous one will change or another one is added in between this test case might start failing